packages/atomic/src/locales.json

```diff
@@ -1,12 +1,41 @@
 {
   "search": {
     "en": "Search",
-    "fr": "Recherche"
+    "fr": "Recherche",
+    "cs": "Hledat",
+    "da": "Søg",
+    "de": "Suchen",
+    "el": "Αναζήτηση",
+    "es-es": "Buscar",
+    "fi": "Hae",
+    "hu": "Keresés",
+    "id": "Cari",
+    "it": "Cerca",
+    "ja": "検索する",
+    "ko": "검색",
+    "nl": "Zoeken",
+    "no": "Søk",
+    "pl": "Wyszukaj",
+    "pt-br": "Pesquisar",
+    "ru": "Искать",
+    "sv": "Sök",
+    "th": "ค้นหา",
+    "tr": "Ara",
+    "zh-cn": "搜索",
+    "zh-tw": "搜尋"
   },
   "searchBox": {
     "en": "Insert a query. Press enter to send.",
     "fr": "Insérer une requête. Appuyez sur Entrée pour envoyer."
   },
+  "facetSearch": {
+    "en": "Search for values in {{label}} facet",
+    "fr": "Rechercher des valeurs dans la facette {{label}}"
+  },
+  "facetValue": {
+    "en": "Inclusion filter on {{value}}; {{numberOfResults}} results",
+    "fr": "Filtre d'inclusion sur {{value}}; {{numberOfResults}} résultats"
```

Comment:
Does `i18n` throw an error if an object not containing the `value` or `numberOfResults` keys is passed?
If not, a unit test could be useful to detect any issues,